starter-code/views/auth/login.hbs

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
```

Comment:
Nice that you practiced using Bootstrap! In the future, try to set it up inside layout.hbs so that way you only need to import the <link> and <script> tags in one place. You can also add the navigation bar there so that it appears automatically on all your views